typescript-lite/generator/src/main/java/org/coursera/courier/tslite/TSSyntax.java

```diff
@@ -139,7 +139,7 @@ public TSSyntax(TSProperties TSProperties) {
 
   /** Different choices for how to escaping symbols that match reserved ts keywords. */
   private static enum EscapeStrategy {
-    /** Adds an underscore after the symbol name when escaping. e.g.: class becomes class_*/
+    /** Adds an underscore after the symbol name when escaping. e.g.: class becomes class$*/
```

Comment:
:man_facepalming: 

Gonna look at that. Also I'm seeing a weird regression regarding docstrings.